bus-mapping/src/circuit_input_builder.rs

```diff
@@ -630,6 +680,23 @@ impl<P: JsonRpcClient> BuilderClient<P> {
             cli: client,
             chain_id: chain_id.into(),
             circuits_params,
+            feature_config: FeatureConfig::default(),
```

Comment:
Maybe this should just call through to `new_with_features` with `FeatureConfig::default()`?

Reply:
Good idea. Fixed in [949c228](https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/1728/commits/949c2284162023737ee90d681fe33d71bd220920)